Makefile

```diff
@@ -1,7 +1,7 @@
 BIN_DIR = bin
 export GOPATH ?= $(shell go env GOPATH)
 export GO111MODULE ?= on
-export PROJECT_SERUM_IMAGE ?= projectserum/build:v0.22.0
+export PROJECT_SERUM_IMAGE ?= projectserum/build:v0.22.1
```

Comment:
Note: `0.23.0` image wasn't released but I think it doesn't matter for builds since the solana version stayed the same. https://hub.docker.com/r/projectserum/build/tags